src/test/java/com/easyrest/tests/LoginTest.java

```diff
@@ -19,11 +14,18 @@ public void testLogin() throws InterruptedException {
         //ExtentSparkReporter spark = new ExtentSparkReporter("src/test/Reports/loginTest_28.06.2022");
         //extent.attachReporter(spark);
 
-        String clientEmail = ConfigProvider.clientEmail;
-        String clientPassword = ConfigProvider.clientPassword;
+        SignUpPage page = new SignUpPage(driver);
```

Comment:
How did you get these changes?

Reply:
I am copied this page  in add_Rest_List branch from main branch because this branch has conflicts. I haven't actually made any changes to it.